### PR TITLE
Update repo

### PR DESCRIPTION
**Pull request by mirii1994**
This PR:
- Upgrade to Filebeat 8.3.2.
- Allow usage with instance IAM Roles.
- Add k8s deployment.

### CODE REVIEW

File: k8s/logzio-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: logzio-mysql-logs
+  labels:
+    k8s-app: logzio-mysql-logs
+rules:
+  - apiGroups:
+    - ""
+    resources:
```
Comment (Doron-Bargo) [inline k8s/logzio-deployment.yaml:20]: Do we really need those rules?

File: k8s/logzio-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  kind: ClusterRole
+  name: logzio-mysql-logs
+  apiGroup: rbac.authorization.k8s.io
+---
+# daemonset
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: logzio-mysql-logs
+  namespace: kube-system
```
Comment (Doron-Bargo) [inline k8s/logzio-deployment.yaml:56]: why we deploy it on kube-system

File: k8s/logzio-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+roleRef:
+  kind: ClusterRole
+  name: logzio-mysql-logs
+  apiGroup: rbac.authorization.k8s.io
+---
+# daemonset
+apiVersion: apps/v1
+kind: DaemonSet
```
Comment (Doron-Bargo) [inline k8s/logzio-deployment.yaml:53]: Why we using daemonSet

File: k8s/logzio-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        #   valueFrom:
+        #     secretKeyRef:
+        #       name: logzio-logs-secret
+        #       key: rds-slow-log-file
+        # - name: RDS_LOG_FILE
+        #   valueFrom:
+        #     secretKeyRef:
+        #       name: logzio-logs-secret
+        #       key: rds-log-file
+        securityContext:
```
Comment (Doron-Bargo) [inline k8s/logzio-deployment.yaml:117]: Do we need to run it as root?

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
Please look on my comments

File: k8s/logzio-deployment.yaml (hunk 3 above)
Comment (mirii1994) [reply k8s/logzio-deployment.yaml]: my bad. fixed.

File: k8s/logzio-deployment.yaml (hunk 2 above)
Comment (mirii1994) [reply k8s/logzio-deployment.yaml:15]: no special reason. You prefer it to be under "monitoring"?

File: k8s/logzio-deployment.yaml (hunk 1 above)
Comment (mirii1994) [reply k8s/logzio-deployment.yaml]: fixed.

File: k8s/logzio-deployment.yaml (hunk 2 above)
Comment (Doron-Bargo) [reply k8s/logzio-deployment.yaml:15]: yes please

File: k8s/logzio-deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+        #       name: logzio-logs-secret
+        #       key: rds-slow-log-file
+        # - name: RDS_LOG_FILE
+        #   valueFrom:
+        #     secretKeyRef:
+        #       name: logzio-logs-secret
+        #       key: rds-log-file
+        resources:
+          limits:
+            memory: 200Mi
```
Comment (Doron-Bargo) [inline k8s/logzio-deployment.yaml:78]: please increase the limit to 1g

File: k8s/logzio-deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+  namespace: kube-system
+  labels:
+    k8s-app: logzio-mysql-logs
+---
+# daemonset
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: logzio-mysql-logs
+  namespace: kube-system
```
Comment (Doron-Bargo) [inline k8s/logzio-deployment.yaml:15]: please replace to Monitoring namespace

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
looking good please look on my comments

File: k8s/logzio-deployment.yaml (hunk 6 above)
Comment (mirii1994) [reply k8s/logzio-deployment.yaml]: done

File: k8s/logzio-deployment.yaml (hunk 5 above)
Comment (mirii1994) [reply k8s/logzio-deployment.yaml]: done

**Review by Doron-Bargo — APPROVED** [review]
lgtm

**Pull request merged**